ts/utils/staking_utils.test.ts

```diff
@@ -0,0 +1,343 @@
+            totalProtocolFeesGeneratedInEth: 0,
+        },
+        nextEpochStats: {
+            poolId: '7',
+            zrxStaked: 0,
+            operatorShare: 1,
+            approximateStakeRatio: 0,
+            makerAddresses: [],
+            totalProtocolFeesGeneratedInEth: 0,
+        },
+    },
+    {
+        poolId: '8',
+        operatorAddress: '0x5409ed021d9299bf6814279a6a1411a7e866a631',
+        createdAt: {
+            blockNumber: 14797017,
+            txHash: '0xb6e261156d6a4a01ced813662e305cfd485d487710f66b259fe7e50009e7db87',
+        },
+        metaData: { isVerified: false },
+        sevenDayProtocolFeesGeneratedInEth: 0,
+        currentEpochStats: {
+            poolId: '8',
+            zrxStaked: 1474.0966666666668,
+            operatorShare: 0,
+            approximateStakeRatio: 0,
+            makerAddresses: [],
+            totalProtocolFeesGeneratedInEth: 0,
+        },
+        nextEpochStats: {
+            poolId: '8',
+            zrxStaked: 1174.0966666666668,
+            operatorShare: 0,
+            approximateStakeRatio: 0,
+            makerAddresses: [],
+            totalProtocolFeesGeneratedInEth: 0,
+        },
+    },
+    {
+        poolId: '9',
+        operatorAddress: '0x5409ed021d9299bf6814279a6a1411a7e866a631',
+        createdAt: {
+            blockNumber: 15303667,
+            txHash: '0x3fe6dc177e40c260bb43716753186a9ba716e097cbac74ac00a9a86d4e360482',
+        },
+        metaData: { isVerified: false },
+        sevenDayProtocolFeesGeneratedInEth: 0,
+        currentEpochStats: {
+            poolId: '9',
+            zrxStaked: 0,
+            operatorShare: 1,
+            approximateStakeRatio: 0,
+            makerAddresses: [],
+            totalProtocolFeesGeneratedInEth: 0,
+        },
+        nextEpochStats: {
+            poolId: '9',
+            zrxStaked: 0,
+            operatorShare: 1,
+            approximateStakeRatio: 0,
+            makerAddresses: [],
+            totalProtocolFeesGeneratedInEth: 0,
+        },
+    },
+    {
+        poolId: '10',
+        operatorAddress: '0x5409ed021d9299bf6814279a6a1411a7e866a631',
+        createdAt: {
+            blockNumber: 15303725,
+            txHash: '0x34de21634f0aee11980ec656b2ea681aa806d5cad380b1dbc9c3aa6ccd139267',
+        },
+        metaData: { isVerified: false },
+        sevenDayProtocolFeesGeneratedInEth: 0,
+        currentEpochStats: {
+            poolId: '10',
+            zrxStaked: 0,
+            operatorShare: 1,
+            approximateStakeRatio: 0,
+            makerAddresses: ['0x5409ed021d9299bf6814279a6a1411a7e866a631'],
+            totalProtocolFeesGeneratedInEth: 0,
+        },
+        nextEpochStats: {
+            poolId: '10',
+            zrxStaked: 0,
+            operatorShare: 1,
+            approximateStakeRatio: 0,
+            makerAddresses: ['0x5409ed021d9299bf6814279a6a1411a7e866a631'],
+            totalProtocolFeesGeneratedInEth: 0,
+        },
+    },
+];
```

Comment:
NIT: might want to dump this in a fixtures folder or similar

Reply:
will do!